api/urls.py

```diff
@@ -24,4 +24,5 @@
     # https://www.django-rest-framework.org/api-guide/versioning/
     path("admin/", admin.site.urls),
     path("api/user/", include("user.urls")),
+    path("api/customer", include("customer.urls"))
```

Comment:
Ideally i want the api endpoint for invoice to just be: `/api/invoices` instead of `/api/customer/invoices` (too lengthy), so just `path("api/", include("customer.urls"))` will do.